.github/workflows/watch.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   generated-code:
+    if: github.repository == 'async-aws/aws'
```

Comment:
doesn't this break the `async-aws-bot` submitting PRs, since it runs on the fork?

Reply:
The Action run on the main repo (https://github.com/async-aws/aws/actions/runs/325544202)
But use the Bot's crendentials to push commits and open pull request on the Fork

Reply:
ah right, thanks